controllers/memberoperatorconfig/mapper.go

```diff
@@ -0,0 +1,30 @@
+package memberoperatorconfig
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type SecretToMemberOperatorConfigMapper struct{}
+
+var _ handler.Mapper = SecretToMemberOperatorConfigMapper{}
+var mapperLog = ctrl.Log.WithName("SecretToMemberOperatorConfigMapper")
+
+// Map maps secrets to the singular instance of MemberOperatorConfig named "config"
+func (m SecretToMemberOperatorConfigMapper) Map(obj handler.MapObject) []reconcile.Request {
+	if _, ok := obj.Object.(*corev1.Secret); ok {
+		controllerNS, err := k8sutil.GetWatchNamespace()
+		if err != nil {
+			mapperLog.Error(err, "could not determine watched namespace")
+			return []reconcile.Request{}
+		}
+		return []reconcile.Request{{NamespacedName: types.NamespacedName{Namespace: controllerNS, Name: "config"}}}
+	}
```

Comment:
IIRC, we watch only in the operator namespace, so you can easily take the namespace from the secret object.
```suggestion
	if secret, ok := obj.Object.(*corev1.Secret); ok {
		return []reconcile.Request{{NamespacedName: types.NamespacedName{Namespace: secret.Namespace, Name: "config"}}}
	}
```

Reply:
if you still want to use the Watch namespace env var - it would be better to store it as a parameter of the `SecretToMemberOperatorConfigMapper` type

Reply:
You're right, we only watch the operator namespace so we can just take the namespace from the secret object.